torch/csrc/jit/codegen/cuda/shape_inference.cpp

```diff
@@ -181,6 +181,54 @@ class NaiveTypePropagator {
         node->output()->setType(out_type);
         break;
       }
+      case aten::native_layer_norm: {
```

Comment:
FYI, In case where a function has overload, use ITS symbol name to identify the function is not safe any more.
But this is safe for us as we only take one of the overloaded function.